core/server/data/migrations/versions/3.0/12-populate-members-table-from-subscribers.js

```diff
@@ -0,0 +1,46 @@
+const ObjectId = require('bson-objectid');
+const _ = require('lodash');
+const membersSchema = require('../../../schema').tables.members;
+const models = require('../../../../models');
+const common = require('../../../../lib/common');
+
+module.exports.config = {
+    transaction: true
+};
+
+module.exports.up = (options) => {
+    const localOptions = _.merge({
+        context: {internal: true},
+        migrating: true
+    }, options);
+
+    const memberAttrs = _.keys(membersSchema);
+
+    return models.Subscribers
+        .forge()
+        .fetch(localOptions)
+        .then(({models: subscribers}) => {
+            if (subscribers.length > 0) {
+                common.logging.info(`Adding ${subscribers.length} entries to subscribers`);
+
+                let members = _.map(subscribers, (subscriber) => {
+                    let member = memberAttrs.reduce(function (obj, entry) {
```

Comment:
If we end up adding columns to the members table which don't exist in subscribers this is gonna break I think? 

It might be best to be explicit about what we're copying here: name, email, created_{at,by} update_{at,by}

Reply:
hm. good point. will update!